enginetest/sqllogictest/convert/convert.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/dolthub/go-mysql-server/enginetest/sqllogictest/utils"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	args := os.Args[1:]
+
+	if len(args) != 1 {
+		panic("expected 1 arg")
+	}
+
+	file, err := os.Open(args[0])
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for {
+		if !scanner.Scan() {
+			break
+		}
+		line := scanner.Text()
+		if len(line) == 0 {
+			fmt.Println()
+			continue
+		}
+		if strings.HasPrefix(line, "#") {
+			fmt.Println(line)
```

Comment:
printing to stdout is a bit harder to control that writing to a bytes.Buffer, and writing to some intentional file name or tempfile at the end